### DISCUSSION TITLE
how to change some wording

### DISCUSSION THREAD

**Discussion by okkhalid** [snipe/snipe-it#9049] [Q&A] [answered]
Hello, 
How I can change the word: State to Province and Zip to Postal Code.
Also how I can remove all other counties and keep only Canada?

https://ibb.co/zsZW4kR

Thanks,

**Comment by snipe** [comment, marked as answer]
You can handle phrases using the `resources/lang/whatever-language` files, but in general we really don't recommend it, as you'll have to keep re-applying those changes every time you upgrade.

**Reply by okkhalid** [reply]
Is there any overwrite style like WordPress - child theme so we do not need to keep re applying. :)
